.circleci/config.yml

```diff
@@ -27,14 +27,14 @@ jobs:
             git submodule init
             git submodule update
       - run: |
-          nix-build -j 4 -A fv3 | cachix push vulcanclimatemodeling
+          nix-build -j 4 -A fms | cachix push vulcanclimatemodeling
+          nix-shell --run "cp -f nix/fv3/configure.fv3 FV3/conf && make -j 4 -C FV3"
```

Comment:
why do you no longer 'nix-build' fv3? I see that you make FV3 in a nix-shell, but it doesn't get pushed? no objection, mostly trying to understand this diff.

Reply:
Sorry, I could have made this cleaner. Now that the packages are being pushed by a2icm/packages, I basically wanted to make the tests as similar to the development workflow as possible.